ginga/util/io_fits.py

```diff
@@ -48,12 +48,14 @@ class FITSError(Exception):
     pass
 
 
-def use(fitspkg, raise_err=True):
+def use(fitspkg, raise_err=True, suppress_verify_warnings=False):
     global fits_configured, fitsLoaderClass
 
     if fitspkg == 'astropy':
         if have_astropy:
             fitsLoaderClass = AstropyFitsFileHandler
+            if suppress_verify_warnings:
+                warnings.simplefilter('ignore', pyfits.verify.VerifyWarning)
```

Comment:
I wonder if this is the right place for warning suppression.  Do we have to move this close to the code that loads the file and use `warnings.catch_warnings` context manager?

1. This PR does not seem to suppress anything for me.
2. The way it is currently done, it is not undone until the session ends, which is probably okay for the standalone app but might be problematic for notebooks (i.e., warnings ignored outside of Ginga but within the same Python session).